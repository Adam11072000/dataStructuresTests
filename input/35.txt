init 4242
addImage 5502
deleteImage 5502
addImage -6
addImage 1097
addImage -9
mergeSuperPixels 1097 1253 0
addImage -4
addImage -5
addImage 1332
addImage 1097
deleteImage 1097
setLabelScore 1332 3635 2900 5747
addImage -9
getHighestScoredLabel 1332 3636
addImage 371
quit