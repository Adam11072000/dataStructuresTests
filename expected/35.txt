init done.
addImage: SUCCESS
deleteImage: SUCCESS
addImage: INVALID_INPUT
addImage: SUCCESS
addImage: INVALID_INPUT
mergeSuperPixels: SUCCESS
addImage: INVALID_INPUT
addImage: INVALID_INPUT
addImage: SUCCESS
addImage: FAILURE
deleteImage: SUCCESS
setLabelScore: SUCCESS
addImage: INVALID_INPUT
getHighestScoredLabel: FAILURE
addImage: SUCCESS
quit done.
